lib/fs.js

```diff
@@ -2009,30 +2009,67 @@ function ReadStream(path, options) {
   this.closed = false;
 
   if (this.start !== undefined) {
-    if (typeof this.start !== 'number' || Number.isNaN(this.start)) {
+    // Case 1: If the type of start is not 'number', throw ERR_INVALID_ARG_TYPE
+    // (TypeError).
```

Comment:
To me these comments seem redundant. It should be clear from the code what happens. So I personally would remove all added comments. What do others think about that?

Reply:
@BridgeAR sure, will do.